test/test_recordings_filter.py

```diff
@@ -85,8 +85,14 @@ def test_recordfing_query_tag_fields(self, helper):
             assert tracks is not None
             for track in tracks:
                 track_tags = track.get("TrackTags")
```

Comment:
This line belongs underneath the start_s and end_s asserts

Reply:
> Have you checked that this works with all parts of the cacophony-browse that use these APIs?

Have run through with browse, shouldn't cause any problems as it's only returning more than it was before